GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
@Trott Everything that you mentioned above seems like it would be very useful to clarify what's expected from CTC members and collaborators. As far as I know, that information is not present in any of the organization's repositories, and so it can be left to interpretation.

I agree that I don't think  that we want to go into details such as:

> Is it enough to "attend" the weekly meeting, or should one be expected to "participate" in the meeting?) 

and that's not what I was suggesting.

For instance, my perception of the role and responsibilities of CTC members is maybe biased by how the pre-foundation node "core team" was operating: it was expected that core team members attend core team meetings, but some members would not attend almost all of them. However because there was no distinction between collaborators and members of the core team, they were still considered to fulfill their responsibilities as contributors with a lot of expertise in different areas.

Writing down a shared understanding of what precisely the role of a CTC member is seems like it would only help everyone understand their responsibilities. Not doing so leaves a lot of room for interpretation, and thus confusion.